src/Game/AI.cpp

```diff
@@ -3,12 +3,234 @@
 //
 
 #include "AI.h"
-namespace aiTools{
-    int evalState(){
-        int val = 0;
+#include <SopraGameLogic/GameModel.h>
+#include <SopraGameLogic/GameController.h>
 
-        //TODO: Make this a useful function
+namespace aiTools {
+    double evalState(const std::shared_ptr<gameModel::Environment> env, bool isLeft, bool goalScoredThisRound) {
```

Comment:
der shared_ptr sollte vom typ const gameModel::Environment sein. Damit stellst du sicher, dass du das übergebene env nicht veränderst
```suggestion
    double evalState(const std::shared_ptr<const gameModel::Environment> &env, bool isLeft, bool goalScoredThisRound) {
```